modules/programs/fish.nix

```diff
@@ -43,7 +43,7 @@ in
           Whether fish should use completion files provided by other packages.
         '';
       };
-      
+
```

Comment:
If the whitespace churn is undesirable, I can remove it or move it to a different commit.